src/mainwindow.cpp

```diff
@@ -4228,7 +4228,8 @@ void MainWindow::on_bossButton_clicked()
   std::string reportURL;
   std::string errorMessages;
 
-  m_OrganizerCore.currentProfile()->writeModlistNow();
+  //m_OrganizerCore.currentProfile()->writeModlistNow();

```

Comment:
The old code is stored in version control (ie., git), so it's only clutter in the current code. Just remove it.